### DISCUSSION TITLE
BG graph not showing on lock screen after March Pixel update

### DISCUSSION THREAD

**Discussion by psjostrom** [NightscoutFoundation/xDrip#2738] [General]
The BG graph is not being displayed on lock screen, or on the notification pulldown menu when the device is locked. It is only shown in the notification pulldown after having unlocked the device.

When did this start: After installing the march pixel update
Device: Pixel 7

![Screenshot_20230317-170827](https://user-images.githubusercontent.com/37512275/225959411-6ffcfec2-09fd-437b-a01e-067ec3314826.png)

With screen locked 
![Screenshot_20230317-170832](https://user-images.githubusercontent.com/37512275/225959472-2ed82a64-00e7-4256-b426-2f22dcd61a60.png)

After unlocking screen 
![Screenshot_20230317-170836](https://user-images.githubusercontent.com/37512275/225959506-5ed93186-8471-4b4c-907f-0b3af93a2c5d.png)

![Screenshot_20230317-171058](https://user-images.githubusercontent.com/37512275/225959633-7c3a905b-227d-4c18-9263-027c36ee5494.png)

**Comment by Navid200** [comment]
Is this only affecting xDrip?

I mean do you have any other app on the same phone that can still show notifications on the locked screen after this same patch?

**Comment by michaeld427** [comment]
There is a new Android option that you need to change to get the graph back.

![Screenshot_20230317-122250](https://user-images.githubusercontent.com/76007398/225962301-d3a6bbfd-b1b6-4dc0-a107-b5a8a8b4a42a.png)

**Reply by psjostrom** [reply]
Thank you, changing it to Show conversations default and silent and solved it for me.

![Screenshot_20230317-173616](https://user-images.githubusercontent.com/37512275/225965246-aba58c1f-9928-4639-83b0-bda3d306603f.png)
